helm/chart/maesh/templates/mesh/mesh-daemonset.yaml

```diff
@@ -64,8 +64,16 @@ spec:
             - "--providers.rest"
             - "--providers.rest.insecure"
             {{- if .Values.tracing.jaeger.enabled }}
+              {{- if .Values.tracing.jaeger.localagenthostport }}
             - "--tracing.jaeger.localagenthostport={{ .Values.tracing.jaeger.localagenthostport }}"
+              {{- else }}
+            - "--tracing.jaeger.localagenthostport=jaeger-agent.maesh.svc.cluster.local:6831"
```

Comment:
Hi, IMHO if you install maesh in another namespace than the default one, you should replace this line with something like : 
            - "--tracing.jaeger.localagenthostport=jaeger-agent.{{ .Release.Namespace }}.svc.cluster.local:6831"